### PR TITLE
Backend APIs for admin

### PR DESCRIPTION
**Pull request by jhanaviB**
Backend APIs for admin

### CODE REVIEW

File: backend/routes/user.js [hunk 1]
```diff
@@ -185,7 +187,10 @@ router.post('/login', async (req, res) => {
     try {
         const loggedInUser = await userService.login(user_id, password);
         console.log(loggedInUser)
-        if (loggedInUser) {
+        if (loggedInUser=="incorrect") {
```
[the remark this answers is not in the document]
Comment (jhanaviB) [reply backend/routes/user.js:190]: While logging in, if the username exists but the password is wrong there is a popup that says "Username/Password combination incorrect!".Check is for that.

[the remark this answers is not in the document]
Comment (MansiGit) [reply backend/routes/user.js:190]: Oh,  can we try to return something else instead? This check is quite confusing. What happens when you return null ?

[the remark this answers is not in the document]
Comment (314shreya) [reply backend/routes/user.js:190]: Yes, return -1

[the remark this answers is not in the document]
Comment (MansiGit) [reply backend/routes/user.js:190]: @jhanaviB We should return null in all cases other than when username and password match. So return null in this case.

**Pull request closed without merging**